processors/filtering/column_filter.py

```diff
@@ -93,9 +95,43 @@ def get_options(cls, parent_dataset=None, user=None):
 
     def process(self):
         """
-        Reads a CSV file, filtering items that match in the required way, and
+        Reads a file, filtering items that match in the required way, and
         creates a new dataset containing the matching values
         """
+        # Filter posts
+        matching_posts = self.filter_items()
+
+        # Write the posts
+        num_posts = 0
+        if self.dataset.get_extension() == "csv":
```

Comment:
Duh, the file does not exist so there is no extension yet. I think the processor.get_extension() should return the parent dataset extension if desired. I can overwrite this method in the column_filter though I think this is related to the pipeline issue as well.